unstable/wlr-exec-secure-unstable-v1.xml

```diff
@@ -0,0 +1,139 @@
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice (including the next
+    paragraph) shall be included in all copies or substantial portions of the
+    Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+  </copyright>
+
+  <description summary="a protocol for executing clients with higher permissions">
+    This protocol allows unprivileged clients to ask the compositor to execute
+    a program which can access protected Wayland protocols.
+
+    The rationale for this protocol is that the execution environment of an
+    arbitrary client cannot be reliably verified by the compositor. Environment
+    variables like LD_PRELOAD may alter the runtime behavior of the client to
+    malicious ends. However, if the compositor creates the client socket and
+    exec's the client program itself, it can be sure of the client's execution
+    environment.
+
+    If a client is run with the WAYLAND_SOCKET environment variable set, it can
+    assume that the socket represents an authenticated Wayland connection and
+    it will be able to use any secure protocols it is afforded. If not, the
+    client will likely connect to the Unix socket specified by WAYLAND_DISPLAY,
+    but it will be an unauthenticated connection. If it needs authenticated
+    protocols to accomplish its basic tasks, it will likely wish to use
+    wlr_exec_secure_unstable_v1 to ask the compositor to execute itself again.
+    It's at the client's discretion to exit once the compositor runs the second
+    process or to wait for it to complete.
+  </description>
+
+  <interface name="zwlr_exec_manager_v1" version="1">
+    <description summary="interface for requesting execution handles">
+      Clients which want the compositor to execute another client should create
+      a zwlr_exec_handle_v1 and use it to configure and run the client.
+    </description>
+
+    <request name="exec_process">
+      <description summary="creates a wlr_exec_handle_v1 resource">
+        Creates an exec_handle for the requested command. The compositor can
+        choose to disallow this request for certain commands, in which case a
+        protocol error is issued. The command will be executed without a shell.
```

Comment:
It should probably be explicit that the arguments are passed unmodified to execv, instead of just saying "executed without a shell".
No $VAR or ~ expansion, etc.